packages/dapp/src/helpers/queryTransformations.ts

```diff
@@ -95,16 +121,47 @@ export function transformGraphQLDataIntoChallenge(queryChallengeData: any): Chal
         votesAgainst: new BigNumber(queryChallengeData.poll.votesAgainst),
       },
       requestAppealExpiry: new BigNumber(queryChallengeData.requestAppealExpiry),
-      appeal: {
-        requester: "0x0",
-        appealFeePaid: new BigNumber(0),
-        appealPhaseExpiry: new BigNumber(0),
-        appealGranted: false,
-        appealOpenToChallengeExpiry: new BigNumber(0),
-        appealChallengeID: new BigNumber(0),
-        appealTxData: undefined,
-        appealChallenge: undefined,
-        statement: undefined,
+      appeal: transformGraphQLDataIntoAppeal(queryChallengeData.appeal),
+    };
+  } else {
+    return undefined;
+  }
+}
+
+export function transformGraphQLDataIntoAppeal(queryAppealData: any): AppealData | undefined {
+  if (queryAppealData) {
+    return {
+      requester: queryAppealData.requester,
+      appealFeePaid: new BigNumber(10), // queryAppealData.appealFeePaid
```

Comment:
Is field just missing from the GraphQL response / the crawler isn't picking it up yet?